docs/docs/reference/dropped-features/delayed-init.md

```diff
@@ -19,8 +18,9 @@ some JVM's means that it will only be interpreted. So, better not use it
 for benchmarking! Also, if you want to access the command line arguments,
 you need to use an explicit `main` method for that.
 ```scala
-object Hello {
-  def main(args: Array[String]) =
-    println(s"Hello, ${args(0)}")
-}
+object Hello:
+   def main(args: Array[String]) =
+      println(s"Hello, ${args(0)}")
 ```
+On the other hand, Scala 3 offers a convenient alternative to such "program" objects
+with [@main methods](../changed-features/main-functions.html).
```

Comment:
Nice addition 👍 

Link to markdown file for consistency?

Reply:
It's @odersky  addition (visible before rebasing with upstream).